src/test/kotlin/com/atlassian/performance/tools/virtualusers/LoadTestTest.kt

```diff
@@ -88,10 +91,13 @@ class LoadTestTest {
         synchronized(globalStateLock) {
             TestBrowser.reset()
             TracingScenario.reset()
+            HardcodedUserGenerator.reset()
             loadTest.run()
         }
 
-        assertThat(userGenerator.usersCreated.get()).isEqualTo(5)
+
+
```

Comment:
Let's avoid these blank lines.
Blank lines have meaning in tests. They demarcate the given-when-then sections.